sdk/servicebus/service-bus/src/util/retries.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Constants, RetryOptions } from "@azure/core-amqp";
+
+/**
+ * @internal
+ * @ignore
+ */
+// Copied from event-hubs
+// TO DO - move this retry-utils to core-amqp
+export function getRetryAttemptTimeoutInMs(retryOptions: RetryOptions | undefined): number {
```

Comment:
unit tests for this?

Reply:
Moved it to core-amqp

Reply:
Moved it to core-amqp, you want me to add tests over there?

Reply:
Yes, definitely :)